lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -147,7 +156,7 @@ <h3 class="course-title">
                 % endif
               % else:
                 % if not is_course_blocked:
-                  <a href="${course_target}" class="enter-course" data-course-key="${enrollment.course_id}">${_('View Course')}<span class="sr">&nbsp;${course_overview.display_name_with_default}</span></a>
+                  <a href="${course_target}" class="enter-course ${'hidden' if is_unfulfilled_entitlement else ''}" data-course-key="${enrollment.course_id}">${_('View Course')}<span class="sr">&nbsp;${course_overview.display_name_with_default}</span></a>
```

Comment:
The screenreader text may not be positioned correctly in foreign languages. It should be part of the gettext call.

Reply:
Same as above